### ISSUE TITLE
#create method consistency

### ISSUE THREAD

**Issue by KokoNeot** [discordjs/discord.js#8261]
### Which package is the feature request for?

discord.js

### Feature

I'd like make a suggestion to make GuildChannelManager#create method consistent with RoleManager#create method.
the GuildChannelManager#create takes 2 arguments: name and options while RoleManager#create takes 1 argument: options containing "name" property.
If this is on purpose, could somebody explain to me why is it like this? 

Also could I please be unblocked from this repository @iCrawl on my [account](https://github.com/DaliborTrampota)? It's been an year already and I've reported few bugs. I've learned from my mistakes and I apologized. Thank you for consideration.

note: when writing this I noticed that the type names are also not consistent: RoleCreateOptions and GuildChannelCreateOptions but thats just minor thing, the create method inconsistency is quite annoying though

### Ideal solution or implementation

Make GuildChannelManager#create take only one parameter, CreateChannelOptions which will also include "name" property
 

### Alternative solutions or implementations

There is really no alternative solution apart from just using it how it is now

**Comment by ImRodry** [comment]
Yeah this was proposed but the PR was so bad it got closed https://github.com/discordjs/discord.js/pull/7448
I think it would be a good idea if done properly

**Comment by almeidx** [comment]
Already done in #8026

**Comment by ImRodry** [comment]
Ah oops

**Comment by iCrawl** [comment]
> I'd like make a suggestion to make GuildChannelManager#create method consistent with RoleManager#create method.
the GuildChannelManager#create takes 2 arguments: name and options while RoleManager#create takes 1 argument: options containing "name" property.
If this is on purpose, could somebody explain to me why is it like this?

Is this meant for version 13 or something? Because on v14/main it already is like that: https://discord.js.org/#/docs/discord.js/main/class/GuildChannelManager?scrollTo=create

Also seen in the list in #8026

**Comment by KokoNeot** [comment]
okay great. yes it was v13 request. I'm not on dev so didnt know it was already implemented in there.
